product/roundhouse.console/app.config

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <system.data>
+    <DbProviderFactories>
+      <add name="Npgsql" invariant="Npgsql"
+      description=".Net Framework Data Provider for Postgres"
+      type="Npgsql.NpgsqlFactory, Npgsql, Version=3.0.3.0, Culture=neutral, PublicKeyToken=5d8b90d52f46fda7"/>
+    </DbProviderFactories>
+  </system.data>

```

Comment:
This is what is required to use https://github.com/chucknorris/roundhouse/blob/master/product/roundhouse/databases/AdoNetDatabase.cs#L31. We are telling `System.Data.DbProviderFactories` that we have a factory called `Npgsql` (which comes from https://github.com/chucknorris/roundhouse/blob/master/product/roundhouse.databases.postgresql/PostgreSQLDatabase.cs#L86). Npgsql implements the required interface.

To my knowledge, this would need to be done for all other database types. You can see something similar for MySql here - https://github.com/chucknorris/roundhouse/blob/master/product/roundhouse.databases.mysql/MySqlDatabase.cs#L81

Oracle: https://github.com/chucknorris/roundhouse/blob/master/product/roundhouse.databases.oracle/OracleDatabase.cs#L87

Some can be registered in the machine config by installers, in this case I'm just adding it for just this application.